{
  "parameter(s)": [
    24,
    68
  ],
  "test_list_of_ones": [
    "passed",
    "",
    0.00017
  ],
  "test_list_of_prime_numbers": [
    "failed",
    "RecursionError: maximum recursion depth exceeded while calling a Python object",
    0.00156
  ]
}